src/pyannote/audio/__main__.py

```diff
@@ -40,15 +40,24 @@
 import torch
 import typer
 import yaml
-from pyannote.audio import Audio, Pipeline, Model
+from meeteval.io.seglst import SegLST, SegLstSegment
 from pyannote.core import Annotation
 from pyannote.metrics.base import BaseMetric
 from pyannote.metrics.diarization import DiarizationErrorRate, JaccardErrorRate
+from pyannote.metrics.transcription import (
```

Comment:
Let's make it work also when `pyannote.metrics.transcription` fails at import time.